Extension/src/LanguageServer/client.ts

```diff
@@ -3452,6 +3462,25 @@ export class DefaultClient implements Client {
                     line: range.end.line
                 }
             },
+            formatParams: {
+                editorConfigSettings: {},
```

Comment:
You should call getEditorConfigSettings() with the path to the file in question to read the vcFormat settings values that apply to the file in question, and populate editorConfigSettings here.

Reply:
It looks like you want to call `settings.useVcFormat(editor.document)` first.  If the formatting setting is set to "default", this will check if an editorConfig file is present with vcFormat entries in it.  If so, it will return true. (It'll also return true if formatting is set to vcFormat).  If true, you'll want to call `getEditorConfigSettings(editor.document.uri.fsPath)` to include relevant vcFormat settings in the request.